src/common/src/array/data_chunk.rs

```diff
@@ -56,6 +57,40 @@ impl From<usize> for Vis {
     }
 }
 
+impl Vis {
+    pub fn is_empty(&self) -> bool {
+        match self {
+            Vis::Bitmap(b) => b.is_empty(),
+            Vis::Compact(c) => *c == 0,
+        }
+    }
+
+    pub fn len(&self) -> usize {
+        match self {
+            Vis::Bitmap(b) => b.len(),
+            Vis::Compact(c) => *c,
+        }
+    }
```

Comment:
* Shall we call it `capacity`?
* Do we expect this `Vis` to expose the same interface as `Bitmap`? Essentially they are the same thing. Or maybe we should just refactor Bitmap to use either a compact usize or a Bytes.

Reply:
1. I'd prefer len, `capacity` will bring some ambiguity.
2. Generally I think yes, but it's a huge refactor.